tests/unit/business-lookup-services.spec.ts

```diff
@@ -20,11 +20,11 @@ describe('Business Lookup Services', () => {
     // mock successsful search
     sinon.stub(axios, 'get')
       .withArgs('https://search.api.url/businesses/search/facets?start=0&rows=20&categories=legalType:' +
-        'BC,A,ULC,C,S,XP,GP,LP,CUL,XS,LLC,LL,BEN,CP,CC,XL,FI,XCP,PA::status:ACTIVE&query=value:FM1000002')
+        'SP::status:ACTIVE&query=value:FM1000002')
       .returns(new Promise(resolve => resolve({ data: { searchResults: { results: [result] } } })))
 
     // search and look at results
-    const results = await BusinessLookupServices.search('FM1000002', 'ACTIVE')
+    const results = await BusinessLookupServices.search('FM1000002', 'ACTIVE', 'SP')
```

Comment:
Just some simplifications so we don't have to carry around the whole list of legal types multiple times.